### PR TITLE
LIA-56753 Update babel preset

### PR DESCRIPTION
**Pull request by jaker**

### CODE REVIEW

**Comment by jaker** [comment]
To test, switch local installation of `lithium-sdk` to branch `feature/LIA-56753 `. 
Run `npm update`
Test gulp tasks

File: package.json [hunk 1]
```diff
@@ -65,7 +66,7 @@
     "karma-commonjs": "0.0.8",
     "karma-coverage": "0.2.6",
     "karma-growl-reporter": "0.1.1",
-    "karma-jasmine": "^0.1.0",
+    "karma-jasmine": "^0.1.6",
```
Comment (dougschroeder) [inline package.json]: Could we (and should we) move the test-related dependencies under `devDependencies`?

Comment (jaker) [reply package.json]: Normally, that's exactly where these sorts of packages should go. I thought we had `devDependencies` at one point but we might have gotten rid of them. 

@adamayres do you have an objection if I move the karma pkgs to `devDependencies`?

Comment (adamayres) [reply package.json]: No objections.

**Pull request merged**